tests/time_filter_fast_test.py

```diff
@@ -1,47 +1,29 @@
-import unittest
+from traveltimepy import LocationId
+from traveltimepy.dto.requests.time_filter_fast import Transportation
 
-from unittest import mock
 
-from pydantic.tools import parse_raw_as
+def test_one_to_many(sdk, locations):
+    results = sdk.time_filter_fast(
+        locations=locations,
+        searches={
+            LocationId('London center'): [LocationId('Hyde Park'), LocationId('ZSL London Zoo')],
```

Comment:
I think this would look more clear if we just provide strings

Reply:
sure, I just took this idea from scala 🙃

Reply:
That's how it looks :)

Reply:
removed

Reply:
Thanks